src/openbiolink/graph_creation/graphCreation.py

```diff
@@ -71,30 +66,27 @@ def __init__(self, folder_path, use_db_metadata_classes=None, use_edge_metadata_
 
     # ----------- download ----------
 
-    def download_db_files(self):
-        skip = None
-        for_all = False
-        if not globalConfig.INTERACTIVE_MODE:
-            skip = globalConfig.SKIP_EXISTING_FILES
-            for_all = True
-        if not os.path.exists(gcConst.O_FILE_PATH):
-            os.makedirs(gcConst.O_FILE_PATH)
+    def download_db_files(
+        self, skip_existing: bool = True,
```

Comment:
The logic with all of these global constants was too hard to understand... so I took it out. Maybe it would be best to switch all of these constants to arguments to this function, or as members of the class. What do you think @nomisto ?

Reply:
Yes the globalConfig variables could definitely be switched to arguments, as a lot of constants are only used in one or only a few functions/classes. This would probably increase the readability a lo. I'm not really sure what you mean by "this function" (I assume its a typo) as the other constants in the globalConfig are used elsewhere (f.e. the COL_NAMES_* in the trainTestSplitCreation)